style.css

```diff
@@ -1,3 +1,100 @@
+body {
+  font-family: "Quattrocento Sans", sans-serif;
+  text-align: center;
+  color: rgb(90, 90, 90);
+  padding: 7vh 4vw;
+  background-color: rgb(233, 240, 245);
+}
+
+/* ------------ Title and logo -------------- */
+
+h1 {
+  color: rgb(90, 90, 90);
+  letter-spacing: 0.2rem;
+  text-transform: uppercase;
+  margin-bottom: 50px;
+}
+
+#index-img {
+  width: 70%;
+  max-width: 300px;
+  height: auto;
+  margin-bottom: 50px;
+  background-color: rgba(255, 255, 255, 0.863);
+  border: 7px solid rgb(185, 221, 198);
+  border-radius: 50%;
+}
+
+/* ------------ Search section -------------- */
+#get-country {
+  max-width: 800px;
+  min-width: 300px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0;
+}
+
+#search-section {
+  max-width: 800px;
+  min-width: 300px;
+  height: auto;
+  min-height: 150px;
+  padding-top: ;
+  background-color: rgb(185, 221, 198);
```

Comment:
Great color choice! I would put the colors in a variable, so we could reuse them in the project.

Reply:
Good idea!